tests/ObfuscateVar/basic.c

```diff
@@ -0,0 +1,15 @@
+//RUN : clang -Xclang -load -Xclang ./LLVMObfuscateVar.so  -S basic.c -emit-llvm -O2 -o basic.ll
```

Comment:
you should add a check here to ensure everything was done correctly. Run the program and check its output vs. the original output. And also grep the bytecode and count the number of add to verify the split was correct.